### ISSUE TITLE
TypeError: request() got an unexpected keyword argument 'encode_chunked'

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-gui#2793]
https://sentry.io/organizations/gns3/issues/510330847/?referrer=github_plugin

```
TypeError: request() got an unexpected keyword argument 'encode_chunked'
(5 additional frame(s) were not displayed)
...
  File "urllib/request.py", line 526, in open
    response = self._open(req, data)
  File "urllib/request.py", line 544, in _open
    '_open', req)
  File "urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "urllib/request.py", line 1346, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "urllib/request.py", line 1318, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))

TypeError: request() got an unexpected keyword argument 'encode_chunked'
```

**Comment by grossmj** [comment]
https://www.gns3.com/community/discussion/python-3-6-typeerror-request-got

**Comment by grossmj** [comment]
Fixed.

**Comment by grossmj** [comment]
Looks like the problem is still present in version 2.1.18

**Comment by grossmj** [comment]
Sentry issue: [GNS3-GUI-1D5](https://sentry.io/organizations/gns3/issues/865491919/?referrer=github_plugin)

**Comment by grossmj** [comment]
I have fixed this on the 2.2 branch, will probably backport it to 2.1

**Comment by grossmj** [comment]
Backport done.
